parser/transcript/src/test/java/de/danoeh/antennapod/parser/transcript/PodcastIndexTranscriptParserTest.java

```diff
@@ -0,0 +1,157 @@
+package de.danoeh.antennapod.parser.transcript;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+import de.danoeh.antennapod.parser.transcript.PodcastIndexTranscriptParser;
+import de.danoeh.antennapod.model.feed.Transcript;
+
+@RunWith(RobolectricTestRunner.class)
+public class PodcastIndexTranscriptParserTest {
+    private static String srtStr = "1\n"
```

Comment:
Could you please split this up into two tests, one for SRT and one for JSON?

Reply:
done